config/application.rb

```diff
@@ -0,0 +1,32 @@
+require_relative "boot"
+
+require "rails/all"
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module HelloRailsBackEnd
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.1
+
```

Comment:
Nice job so far😉. Currently, your backend has a bug generated from the CORS policy, which is preventing your application from running👍🏼. Kindly enable cors in your application by adding this block of code in your config/application.rb file, inside the Application class as shown below😉.

```suggestion
    config.middleware.insert_before 0, Rack::Cors do
      allow do
        origins 'http://localhost:3001'
        resource '*', headers: :any, methods: [:get, :post, :put, :patch, :delete, :options, :head], credentials: true
      end
    end
```